pkg/didcomm/protocol/introduce/models.go

```diff
@@ -7,18 +7,20 @@ SPDX-License-Identifier: Apache-2.0
 package introduce
 
 import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/decorator"
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/didexchange"
 )
 
 // Proposal defines proposal request
 type Proposal struct {
-	Type   string            `json:"@type,omitempty"`
-	ID     string            `json:"@id,omitempty"`
-	To     *To               `json:"to,omitempty"`
-	NWise  bool              `json:"nwise,omitempty"`
-	Thread *decorator.Thread `json:"~thread,omitempty"`
-	Timing *decorator.Timing `json:"~timing,omitempty"`
+	service.Metadata `json:",squash"`  // nolint: staticcheck
```

Comment:
The protocol's models should only have properties that travel over the wire.

Reply:
Done!